Makefile

```diff
@@ -0,0 +1,24 @@
+image_tor := patrickod/tor
+
+.PHONY: start stop run build tor
+
+start:
+	docker start tor
+
+stop:
+	docker stop tor
+
+run: tor
+
+tor:
+	-@docker rm -f "$@"
+	docker run -d \
+		--name "$@" \
+		-e "TZ=Europe/Berlin" \
```

Comment:
I hope that users which use the Makefile will look in it first and read the comments which I just added. If you would like to take it out, please do so.